cime/config/e3sm/testmods_dirs/bench/gmpas_noio/user_nl_mpaso

```diff
@@ -0,0 +1,2 @@
+config_am_highfrequencyoutput_enable=false
+config_am_globalstats_enable=false
```

Comment:
@amametjanov - there are other output streams you could turn off, if you want. But I think these are the two most expensive ones...